### PR TITLE
fix(discovery): update order of the tvl, little bug fixing

### LINKED ISSUE #928: Clicking on the name of a DAO in discovery takes user to a broken page. Should be the DAOs page on Astro
**Describe the bug**
When user clicks on the name of a DAO in the Discovery section they are taken to a new page inside of discovery without any data. It seems broken and is confusing. Instead users should be taken to the DAOs page on Astro, for example https://app.astrodao.com/dao/auroradao.sputnik-dao.near

**Screenshots**

https://user-images.githubusercontent.com/408953/163494982-9c4c36a2-8732-4662-b02c-f91074057c6a.mp4

### PR DESCRIPTION
**Pull request by stee1house**

### CODE REVIEW

File: astro_2.0/features/Discover/components/Flow/Flow.tsx [hunk 1]
```diff
@@ -118,6 +118,13 @@ export const Flow: FC = () => {
     [isMounted]
   );
 
+  useEffect(() => {
+    setLeaderboardData(null);
+    setOffset(0);
+    setTotal(0);
+    setActiveView(items[0].id);
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Discover/components/Flow/Flow.tsx]: Can it be that items is an empty array

File: astro_2.0/features/Discover/components/GeneralInfo/GeneralInfo.tsx [hunk 2]
```diff
@@ -65,6 +65,13 @@ export const GeneralInfo: FC = () => {
   }, [generalData?.activity.count, generalData?.activity.growth, query.dao, t]);
   const [activeView, setActiveView] = useState(items[0].id);
 
+  useEffect(() => {
+    setLeaderboardData(null);
+    setOffset(0);
+    setTotal(0);
+    setActiveView(items[0].id);
+  }, [items, query.dao]);
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Discover/components/GeneralInfo/GeneralInfo.tsx]: is there sense to move it to a hook and reuse?

File: astro_2.0/features/Discover/components/Flow/Flow.tsx (hunk 1 above)
Comment (stee1house) [reply astro_2.0/features/Discover/components/Flow/Flow.tsx]: No, it is always a predefined array.

File: astro_2.0/features/Discover/components/GeneralInfo/GeneralInfo.tsx (hunk 2 above)
Comment (stee1house) [reply astro_2.0/features/Discover/components/GeneralInfo/GeneralInfo.tsx]: Fixed

**Review by AlexeyPonomarenko — DISMISSED** [review]

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**